<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>374</width>
    <height>314</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 237, 208)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListView" name="stackListView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>65</y>
      <width>171</width>
      <height>131</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255)</string>
    </property>
   </widget>
   <widget class="QListView" name="queueListView">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>65</y>
      <width>171</width>
      <height>131</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="peekStackButton">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>250</y>
      <width>171</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>7</pointsize>
      <italic>false</italic>
      <bold>false</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(21, 0, 0);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>LATEST FINISHED ORDER</string>
    </property>
   </widget>
   <widget class="QPushButton" name="popButton">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>228</y>
      <width>171</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>7</pointsize>
      <italic>false</italic>
      <bold>false</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(21, 0, 0);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>DELETE LATEST ORDER</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>206</y>
      <width>171</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>7</pointsize>
      <italic>false</italic>
      <bold>false</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(21, 0, 0);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>MARK ORDER AS COMPLETE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="enqueueButton">
    <property name="geometry">
     <rect>
      <x>191</x>
      <y>206</y>
      <width>171</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>7</pointsize>
      <italic>false</italic>
      <bold>false</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(21, 0, 0);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>QUEUE ORDER</string>
    </property>
   </widget>
   <widget class="QPushButton" name="dequeueButton">
    <property name="geometry">
     <rect>
      <x>191</x>
      <y>228</y>
      <width>171</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>7</pointsize>
      <italic>false</italic>
      <bold>false</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(21, 0, 0);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>DEQUEUE ORDER</string>
    </property>
   </widget>
   <widget class="QPushButton" name="peekQueueButton">
    <property name="geometry">
     <rect>
      <x>191</x>
      <y>250</y>
      <width>171</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>7</pointsize>
      <italic>false</italic>
      <bold>false</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(21, 0, 0);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>RECENT QUEUE</string>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>351</width>
      <height>21</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>Small Fonts</family>
         <italic>true</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LayoutDirection::LeftToRight</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">color:	rgb(34, 0, 0)</string>
       </property>
       <property name="text">
        <string>COMPLETED ORDERS</string>
       </property>
       <property name="textFormat">
        <enum>Qt::TextFormat::AutoText</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <family>Small Fonts</family>
         <italic>true</italic>
         <fontweight>ExtraBold</fontweight>
        </font>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LayoutDirection::LeftToRight</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">color:	rgb(34, 0, 0)</string>
       </property>
       <property name="text">
        <string>COMPLETED ORDERS</string>
       </property>
       <property name="textFormat">
        <enum>Qt::TextFormat::AutoText</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>374</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
